testfiles/scandir/unknown-shebang

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/with-contenv bashio
+
+echo "hi"
```

Comment:
What we're testing here is the logic in this repository for discovering script files; not the shellcheck functionality itself. So I think it is sufficient to only have a file with a specific shebang merely existing, and then testing that it gets discovered. The test achieves that and I would like to avoid growing that file any bigger.